src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let sum = 0;
```

Comment:
minor: sum name might be a bit confusing here. callCount would be more appropriate